Resources/config/routing.yml

```diff
@@ -10,4 +10,4 @@ netgen_tags_admin_redirects:
 
 netgen_tags_admin:
     resource: "routing/admin.yml"
-    prefix: /tags/admin
+    prefix: /pjax/tags/admin
```

Comment:
Is this absolutely necessary? :)

Tags admin UI can work as as standalone app, so I'd prefer if I don't have to use `/pjax` prefix for that.